src/shared/cache/GenomeNexusMyVariantInfoCache.ts

```diff
@@ -0,0 +1,53 @@
+import * as _ from "lodash";
```

Comment:
@leexgh there'a lot of duplication by making a copy of the cache file like this. There might be a way to instead change the constructor to add a parameter that indicates what fields to add. We can fix it at a later stage if we end up needing more caches